### PR TITLE
refactor(suite): Delete secondary buttons

### PR DESCRIPTION
**Pull request by jvaclavik**
## Related Issue

Resolve

### CODE REVIEW

File: packages/components/src/components/Warning/WarningButton.tsx [hunk 1]
```diff
@@ -6,9 +6,18 @@ export const WarningButton = ({ children, ...rest }: ButtonProps) => {
     const { variant } = useWarningContextContext();
     const value = { variant: DEFAULT_VARIANT };
 
+    const getVariant = () => {
+        switch (variant) {
+            case 'secondary':
+                return 'primary';
```
Comment (peter-sanderson) [inline packages/components/src/components/Warning/WarningButton.tsx]: wut?

File: packages/suite/src/views/wallet/details/index.tsx [hunk 2]
```diff
@@ -107,6 +107,7 @@ const Details = () => {
                                 description={<Translation id={accountTypeDesc} />}
                                 buttonLink={accountTypeUrl}
                             />
+<<<<<<< HEAD
```
Comment (peter-sanderson) [inline packages/suite/src/views/wallet/details/index.tsx]: !!!!

**Comment by seibei-iguchi** [comment]
Why not update the uses of tertiary to secondary and delete tertiary?

File: packages/components/src/components/Warning/WarningIconButton.tsx [hunk 3]
```diff
@@ -6,12 +6,21 @@ export const WarningIconButton = ({ icon, ...rest }: IconButtonProps) => {
     const { variant } = useWarningContextContext();
     const value = { variant: DEFAULT_VARIANT };
 
+    const getVariant = () => {
+        switch (variant) {
+            case 'secondary':
+                return 'primary';
```
Comment (peter-sanderson) [inline packages/components/src/components/Warning/WarningIconButton.tsx]: wat?

**Comment by jvaclavik** [comment]
> Why not update the uses of tertiary to secondary and delete tertiary?

It's going to happen, after this step, but I wanted to make sure there are no secondary ones.

File: packages/components/src/components/Warning/WarningIconButton.tsx (hunk 3 above)
Comment (jvaclavik) [reply packages/components/src/components/Warning/WarningIconButton.tsx]: We don't have anything to pair with. Do you have a better idea?

**Review by peter-sanderson — APPROVED** [review]
image

**Review by seibei-iguchi — APPROVED** [review]

**Pull request merged**